src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_Assessor_Certificates.sql

```diff
@@ -0,0 +1,39 @@
+﻿Create Table [ASData_PL].[Assessor_Certificates]
+(
+[Id]											[uniqueidentifier]		PRIMARY KEY NOT NULL,
+[AchievementDate]								[DateTime2](7)			NULL,
+[BatchNumber]								    [Int]					NULL,
+[CertificateReference]							[NVarchar](50)			NULL,
+[CertificateReferenceId]						[Int]					NULL,
+[ContactName]									[NVarchar](100)			NULL,
+[ContactOrganisation]							[NVarchar](200)			NULL,
+[ContactPostCode]								[NVarchar](20)			NULL,
```

Comment:
Contact Post Code might not be required? again from a Security Perspective

Reply:
It is provider postcode and not the learner postcode, I made it as partial postcode rather than full postcode